tensorflow/lite/micro/kernels/svdf.h

```diff
@@ -46,6 +57,16 @@ void EvalIntegerSvdfReference(TfLiteContext* context, TfLiteNode* node,
                               TfLiteEvalTensor* output_tensor,
                               const OpData& data);
 
+void EvalFloatSVDF(TfLiteContext* context, TfLiteNode* node,
```

Comment:
nit: change name to `EvalFloatSvdfReference`

note Svdf instead of SVDF.

Reply:
Done